atom/browser/browser_context_keyed_service_factories.cc

```diff
@@ -59,6 +60,7 @@ void EnsureBrowserContextKeyedServiceFactoriesBuilt() {
   extensions::StorageFrontend::GetFactoryInstance();
   extensions::WebRequestAPI::GetFactoryInstance();
   extensions::AtomExtensionSystemFactory::GetInstance();
+  extensions::UpdateServiceFactory::GetInstance();
```

Comment:
Ya this was left over from when I did have CreateUpdateClient which I didn't end up needing, removing.